src/main/webapp/resources/js/pages/sequencing-runs/store.js

```diff
@@ -1,12 +1,14 @@
 import { configureStore } from "@reduxjs/toolkit";
 import { sequencingRunsApi } from "../../apis/sequencing-runs/sequencing-runs";
+import { runReducer } from "./services/runReducer";
 
 /*
 Redux Store for sequencing runs.
 For more information on redux stores see: https://redux.js.org/tutorials/fundamentals/part-4-store
  */
 export default configureStore({
   reducer: {
+    reducer: runReducer,
```

Comment:
This all should be on the admin store, there should only be one store

Reply:
After talking, I believe we said that we need to retain this for technician access. A new store and routes were created within the admin panel.
Pleas see 00f9906498a48bb29ff23ed4e76bcf57baaf8d72.